.github/workflows/node.js.yml

```diff
@@ -29,6 +51,12 @@ jobs:
             - name: "Run eslint"
               run: |
                   npm run lint
+            - name: Setup mediawiki-docker
```

Comment:
Setup node and mediawiki-docker in a separate job. This will create a cleaner job dependency chain for us